client/my-sites/stats/checklist-banner/index.jsx

```diff
@@ -110,7 +111,7 @@ export class ChecklistBanner extends Component {
 			return false;
 		}
 
-		const abtests = store.get( 'ABTests' );
+		const abtests = store.get( ABTEST_LOCALSTORAGE_KEY );
```

Comment:
I know this isn't the main thrust of your PR, but this file definitely seems like it shouldn't be relying directly on `store`. Ideally it would rely on an abtest lib

Reply:
Definitely agreed, this is an anti-pattern. I'll add a comment noting it as such, but it'd be even worse if we had just left `'ABTests'` string in there untouched.